contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--4.2.0--4.3.0.sql

```diff
@@ -37,6 +37,10 @@ LANGUAGE plpgsql;
  * So make sure that any SQL statement (DDL/DML) being added here can be executed multiple times without affecting
  * final behaviour.
  */
+SET allow_system_table_mods = ON;
+ALTER TABLE sys.babelfish_function_ext DROP CONSTRAINT babelfish_function_ext_pkey;
```

Comment:
do we have upgrade test with entries in babelfish_function_ext which covers this use case?

Reply:
Yeah several tests in which we are creating functions/procedure, in-fact function/procedures are quite abundant in our upgrade tests so the scenario of catalog dropping primary key when catalog is not empty is being covered.